lib/menu.rb

```diff
@@ -0,0 +1,8 @@
+class Menu
+
+def dish_list
+  {:chicken => 3, :beef => 4, :pizza => 5}
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)